docs/workflows/verify-images.rst

```diff
@@ -0,0 +1,262 @@
+as valid.
+
+Parsing for this json file is strict: unrecognized, duplicated or otherwise invalid fields cause the
+entire file, and usually the entire operation, to be rejected.
+
+For full details and spec refer to https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md
+
+
+Verify images mirrored with its original signature into Pulp Container Registry
+-------------------------------------------------------------------------------
+
+Usecase: Pulp Container Registy contains mirrored container images and its signatures from remote
+registry.
+
+Create remote and specify the url to sync container images from and sigstore url to fetch and store
+locally their corresponding original signatures.
+
+::
+
+        $ http POST https://fluffy.example.com/pulp/api/v3/remotes/container/container/ name=legacy-registry-rh url=https://registry.access.redhat.com upstream_name=ubi8/ubi-micro sigstore=https://access.redhat.com/webassets/docker/content/sigstore include_tags:=[\"8.5\"]
+        HTTP/1.1 201 Created
+        Access-Control-Expose-Headers: Correlation-ID
+        Allow: GET, POST, HEAD, OPTIONS
+        Connection: keep-alive
+        Content-Length: 697
+        Content-Type: application/json
+        Correlation-ID: f488df2618814ba3a8a073c433bdf642
+        Date: Thu, 10 Mar 2022 15:06:02 GMT
+        Location: /pulp/api/v3/remotes/container/container/64da87b6-7bff-4ab5-9b35-7922e3efd352/
+        Referrer-Policy: same-origin
+        Server: nginx/1.20.1
+        Strict-Transport-Security: max-age=15768000
+        Vary: Accept, Cookie
+        X-Content-Type-Options: nosniff
+        X-Frame-Options: DENY
+
+        {
+            "ca_cert": null,
+            "client_cert": null,
+            "connect_timeout": null,
+            "download_concurrency": null,
+            "exclude_tags": null,
+            "headers": null,
+            "include_tags": [
+                "8.5"
+            ],
+            "max_retries": null,
+            "name": "legacy-registry-rh",
+            "policy": "immediate",
+            "proxy_url": null,
+            "pulp_created": "2022-03-10T15:06:02.107728Z",
+            "pulp_href": "/pulp/api/v3/remotes/container/container/64da87b6-7bff-4ab5-9b35-7922e3efd352/",
+            "pulp_labels": {},
+            "pulp_last_updated": "2022-03-10T15:06:02.107769Z",
+            "rate_limit": null,
+            "sigstore": "https://access.redhat.com/webassets/docker/content/sigstore",
+            "sock_connect_timeout": null,
+            "sock_read_timeout": null,
+            "tls_validation": true,
+            "total_timeout": null,
+            "upstream_name": "ubi8/ubi-micro",
+            "url": "https://registry.access.redhat.com"
+        }
+
+After sync task completion, create distribution with the base_path ``local-ubi8-repo`` under which
+the image and signatures will be available for pull and verification:
```

Comment:
I think it is important, that we use the same name, right? We should state so.

Reply:
Actually in this case it is not, since the remapIdentity will handle it.
It's important to use same future_base_bath during signing operation as the base_path  of the distribution.